ibm/service/scc/data_source_ibm_scc_scope.go

```diff
@@ -91,20 +65,15 @@ func DataSourceIbmSccScope() *schema.Resource {
 				Description: "A list of scopes/targets to exclude from a scope.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
-						"account_id": &schema.Schema{
-							Type:        schema.TypeString,
-							Computed:    true,
-							Description: "The ID of the account to exclude. Only works with the attribute properties.0.account_group_id set",
-						},
-						"resource_group_id": &schema.Schema{
+						"scope_type": &schema.Schema{
 							Type:        schema.TypeString,
-							Computed:    true,
-							Description: "The ID of the resource group to exclude. Only works with the attribute properties.0.account_id set",
+							Optional:    true,
+							Description: "A target to exclude from the ibm_scc_scope.",
 						},
-						"account_group_id": &schema.Schema{
+						"scope_id": &schema.Schema{
```

Comment:
Why are we not following the same pattern as the main property using account.resource_group format ?

Reply:
we discussed this in a recorded meeting, and it was agreed that the new attributes are acceptable. Please elaborate why there is a sudden need for a change.

Reply:
this is the a sample response of a scope with exclusions:
```json
...
"name": "exclusions",
      "value": [
        {
          "scope_id": "957934c8d570423299a92a7ca7acd334",
          "scope_type": "account.resource_group"
        }
      ]
```
i do not understand the issue of having it defined in the attribute `exclusions`